scenario.py

```diff
@@ -179,6 +180,14 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_ru
         else:
             self.is_early_stopping = True  # default
 
+        # Model used to initialise model
+        if params["init_model_from"] == "random_initialization":
+            self.init_model_from = params["init_model_from"]
+            self.use_saved_weights = False
+        else:
+            self.init_model_from = params["init_model_from"]
+            self.use_saved_weights = True
+
```

Comment:
In both cases you have `self.init_model_from = params["init_model_from"]`, so wouldn't it be better to have that before the conditional statement?